content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/cloud-account-associations/index.md

```diff
@@ -0,0 +1,250 @@
+
+> _Note: As of this writing, this feature is only available _in preview_. Additionally, setting up AWS account associations through the Chainguard Enforce Console is not available at this time. 
+
+
+### Setting up a GCP Cloud Account Association through the Console
+
+If you clicked the edit icon for GCP, the modal window will outline a four step process to set up a cloud account association.
+
+#### Authentication
+
+First, the Console will ask you to authenticate to GCP. Click the **Authenticate** button, which will open an OpenID Connect login flow. Select the GCP account you want to associate with your Chainguard Enforce IAM group. Following that, Chainguard will be able to create Cloud resources on your behalf.
+
+
+#### Select a Project
+
+In the second step, a dropdown menu will appear with all the projects available for you to choose. Select the project you want to associate with your Chainguard Enforce group and click **Continue**. 
+
+![Screenshot showing Step 2 of the GCP cloud account association process. There is a dropdown menu labeled "Project" with no items selected yet.](gcp_step2.png)
+
+#### Configure Workload Identity Federation
+
+After selecting a project, the next step is to configure workload identity federation. Rather than supplying a service account key — which would provide Chainguard Enforce with unnecessarily broad permissions — workload identity federation allows Enforce to impersonate service accounts living in your Cloud environment.
+
+If this is your first time setting up a cloud account association for this project you will need to explicitly authorize the creation of three things: the IAM Service Account Credentials API, the Workload Identity, and the Workload Identity Provider.
+
+![Screenshot showing Step 3 of the GCP cloud account association process. This picture shows the three workload identity federation resources in a "Not Ready" state. There is a large "Incomplete" warning along with a "Configure" button above the table.](gcp_step3.png)
+
+Click the **Configure** button to set up these resources. Once all of them show a **Ready** status, click the **Continue** button.
+
+#### Configure Service Accounts
+
+Lastly, you need to configure a set of service accounts which will connect with Chainguard, allowing Enforce to perform certain limited functions on your project's resources.
+
+Again, if you haven't already set up the required service accounts on your chosen project then you must click the **Configure** button to do so.
+
+![Screenshot showing Step 4 of the GCP cloud account association process. This picture shows all six service account profiles are in a "Not Ready" state. There is a large "Incomplete" warning along with a "Configure" button above the table.](gcp_step4.png)
+
+Once each of the service accounts are in a **Ready** state, click the **Done** button to finish setting up your cloud account association.
+
+
+## Removing Cloud Account Associations

```

Comment:
I tried to remove the association via Console, but that does not seem to be possible right now?

Reply:
Not now, and I'm not sure whether that's coming in the future. I'll add a note to clarify that it's not currently possible through the UI